bin/startup.sh

```diff
@@ -12,43 +14,83 @@ if grep -q "libapache2-mod-auth-mellon" "${HOME}/Aptfile" ; then
   else
     echo "Exporting xml matadata file..."
     echo "${MELLON_SP_METADATA}" | base64 --decode >> "${MELLON_DIR}/mellon.xml"
-    chmod 644 ${MELLON_DIR}/mellon.xml
+    chmod 644 "${MELLON_DIR}/mellon.xml"
   fi
 
   if [[ -z "${MELLON_SP_CERT}" ]]; then
     echo "WARNING: MELLON_SP_CERT is not set. Mellon may not work!"
   else
     echo "Exporting certificate file..."
     echo "${MELLON_SP_CERT}" | base64 --decode >> "${MELLON_DIR}/mellon.cert"
-    chmod 644 ${MELLON_DIR}/mellon.cert
+    chmod 644 "${MELLON_DIR}/mellon.cert"
   fi
 
   if [[ -z "${MELLON_SP_KEY}" ]]; then
     echo "WARNING: MELLON_SP_KEY is not set. Mellon may not work!"
   else
     echo "Exporting key file..."
     echo "${MELLON_SP_KEY}" | base64 --decode >> "${MELLON_DIR}/mellon.key"
-    chmod 600 ${MELLON_DIR}/mellon.key
+    chmod 600 "${MELLON_DIR}/mellon.key"
   fi
 
   if [[ -z "${MELLON_IDP_METADATA}" ]]; then
     echo "WARNING: MELLON_IDP_METADATA is not set. Mellon may not work!"
   else
     echo "Exporting IdP xml metadata file..."
     echo "${MELLON_IDP_METADATA}" | base64 --decode >> "${MELLON_DIR}/mellon_idp_metadata.xml"
-    chmod 644 ${MELLON_DIR}/mellon_idp_metadata.xml
+    chmod 644 "${MELLON_DIR}/mellon_idp_metadata.xml"
   fi
 fi
 
 # Compiling conf file
 echo "compiling conf file..."
 erb "${HOME}/vendor/apache2/conf/httpd.conf.erb" > "${HOME}/vendor/apache2/conf/httpd.conf"
 
+# Setting MaxRequestWorkers value according to the container size. Formula: total_mem / apache_process_mem_used (roughly 12mb)
+case "${CONTAINER_SIZE}" in
+  "S" ) SERVER_LIMIT="$(echo "256 / ${APACHE_WORKER_SIZE}" | "${HOME}/.apt/usr/bin/bc")"
```

Comment:
Please don't use BC: example

```suggestion
  "S" ) SERVER_LIMIT="$(( 256 *APACHE_WORKER_SIZE} ))"
```

Reply:
Done - all fixed